modules/cxf/src/test/resources/gzip-encoding-conf-httpn.xml

```diff
@@ -24,35 +36,38 @@
                     <spring:bean class="org.mule.runtime.module.cxf.feature.ProxyGZIPFeature"/>
             </cxf:features>
         </cxf:proxy-service>
-        <flow-ref name="copy-properties"/>
```

Comment:
From what i saw running this tests, the only property that was needed was the encoding, that is already being handled.